tests/test_package_addons.py

```diff
@@ -115,3 +115,42 @@ def test_mender_configure(
         setup_tester_ssh_connection.run(
             "test -x /usr/share/mender/inventory/mender-inventory-mender-configure"
         )
+
+    def test_mender_monitor(
```

Comment:
This will have to be guarded, either here, or in the gitlab.ci file, since it's closed source.
I'm not sure what would be nicest here.

Reply:
resolved here: https://github.com/mendersoftware/mender-dist-packages/pull/118#discussion_r698292426

Reply:
Sorry I missed you original comment here.